docker-compose.yaml

```diff
@@ -38,7 +38,7 @@ services:
       - mongo
 
   caddy:
-    image: caddy
+    image: caddy:2.4.3-alpine
```

Comment:
We'll probably have to re-start things in the current online version to deploy this.